### ISSUE TITLE
Images passed to Vision models with low quality parameter, even though high quality was chosen

### ISSUE THREAD

**Issue by sigvardt** [langgenius/dify#4268]
### Self Checks

- [X] This is only for bug report, if you would like to ask a quesion, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### Dify version

0.6.7

### Cloud or Self Hosted

Cloud

### Steps to reproduce

1. Make a workflow with image capabilities and a LLM node with vision capabilities
2. Ensure LLM node is set to High Quality vision
3. Publish and run app
4. Provide image file to the workflow 
5. The request sent to the LLM has the quality parameter set to low, even tough it is defined to be high
Tested across models and providers - issue seems to be persistent

<img width="415" alt="image" src="https://github.com/langgenius/dify/assets/6000238/e8cbc15d-c882-46b6-b546-e278b749c232">

<img width="402" alt="image" src="https://github.com/langgenius/dify/assets/6000238/35c35c06-bbd5-4bd7-82ae-96da13a8d096">

### ✔️ Expected Behavior

- Defined vision quality settings are reflected in the call the to the LLM endpoint

### ❌ Actual Behavior

- Discrepancy between settings and executed call to endpoint

**Comment by kei-yamazaki** [comment]
I encountered the same issue as well. This problem is still not fixed in Dify 0.6.9.
I also tried using both the gpt-4-turbo and gpt-4o models, but the same issue occurs.

This problem is very difficult to notice because it does not produce an error, and it is quite troublesome as it unintentionally reduces the overall accuracy of the workflow (in my case, I only noticed the problem due to the difference in accuracy when using Dify and Vision API).
